slave-setup/worker_installer/slave_update.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
```

Comment:
Shall we add this content into slave_setup.sh itself?

Reply:
I added the content to slave-setup.sh

Reply:
Cool.. then we do not need this script anymore right?

Reply:
yea, we can delete this.